tests/admin_widgets/test_autocomplete_widget.py

```diff
@@ -0,0 +1,91 @@
+from django import forms
+from django.contrib.admin.widgets import AutocompleteSelect
+from django.forms import ModelChoiceField
+from django.test import TestCase, override_settings
+
+from .models import Album, Band
+from .widgetadmin import site
+
+model_admin = site._registry[Album]
+
+
+class AlbumForm(forms.ModelForm):
+    class Meta:
+        model = Album
+        fields = ['band']
+        widgets = {'band': AutocompleteSelect(model_admin)}
+
+
+class NotRequiredBandForm(forms.Form):
+    band = ModelChoiceField(
+        queryset=Album.objects.all(),
+        widget=AutocompleteSelect(model_admin),
+        required=False
```

Comment:
please include a trailing comma in cases like this so if we add more items later, we don't have to modify the line again